docs/OV_Runtime_UG/ShapeInference.md

```diff
@@ -8,95 +8,57 @@
-    
-You can change input shapes multiple times using the `InferenceEngine::CNNNetwork::reshape` and `InferenceEngine::CNNNetwork::setBatchSize` methods in any order.
-If a model has a hard-coded batch dimension, use `InferenceEngine::CNNNetwork::setBatchSize` first to change the batch, then call `InferenceEngine::CNNNetwork::reshape` to update other dimensions, if needed.
-
-Inference Engine takes three kinds of a model description as an input, which are converted into an `InferenceEngine::CNNNetwork` object:
-1. [Intermediate Representation (IR)](../MO_DG/IR_and_opsets.md) through `InferenceEngine::Core::ReadNetwork`
-2. [OpenVINO Model](../OV_Runtime_UG/model_representation.md) through the constructor of `InferenceEngine::CNNNetwork`
-
-`InferenceEngine::CNNNetwork` keeps an `ngraph::Function` object with the model description internally.
-The object should have fully-defined input shapes to be successfully loaded to Inference Engine plugins.
-To resolve undefined input dimensions of a model, call the `CNNNetwork::reshape` method to provide new input shapes before loading to the Inference Engine plugin.
-
-Run the following code right after `InferenceEngine::CNNNetwork` creation to explicitly check for model input names and shapes:
-
-```cpp
-CNNNetwork network = ... // read IR / ONNX model or create from nGraph::Function explicitly
-const auto parameters = network.getFunction()->get_parameters();
-for (const auto & parameter : parameters) {
-    std::cout << "name: " << parameter->get_friendly_name() << " shape: " << parameter->get_partial_shape() << std::endl;
-    if (parameter->get_partial_shape().is_dynamic())
-        std::cout << "ATTENTION: Input shape is not fully defined. Use the CNNNetwork::reshape method to resolve it." << std::endl;
-}
-```
-
-To feed input data of a shape that is different from the model input shape, reshape the model first.
-
-Once the input shape of `InferenceEngine::CNNNetwork` is set, call the `InferenceEngine::Core::LoadNetwork` method to get an `InferenceEngine::ExecutableNetwork` object for inference with updated shapes.
-
-There are other approaches to reshape the model during the stage of <a href="_docs_MO_DG_prepare_model_convert_model_Converting_Model.html#when_to_specify_input_shapes">IR generation</a> or [ov::Model creation](../OV_Runtime_UG/model_representation.md).
-
-Practically, some models are not ready to be reshaped. In this case, a new input shape cannot be set with the Model Optimizer or the `InferenceEngine::CNNNetwork::reshape` method.
+There are other approaches to change model input shapes during the stage of <a href="_docs_MO_DG_prepare_model_convert_model_Converting_Model.html#when_to_specify_input_shapes">IR generation</a> or [ov::Model creation](../OV_Runtime_UG/model_representation.md).
```

Comment:
I experienced the same but it looks like this approach works rn on our site. `openvino_` is added automatically, bot locally I indeed have the same issue. I propose to leave as-is and check with docs-team in case it will fail the same while apploaded.

https://docs.openvino.ai/latest/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model.html#when-to-specify-input-shapes
![image](https://user-images.githubusercontent.com/55839243/158356768-ac205490-2460-4cd0-a4d2-c8257a63b1e5.png)